rdr_service/config/config_prod.json

```diff
@@ -139,10 +141,23 @@
           ]
       }
     },
+    "awardee-arizona@all-of-us-ops-data-api-stable.iam.gserviceaccount.com": {
+      "roles": [
+        "awardee_sa"
+      ],
+      "awardee": "arizona",
```

Comment:
Looks like all other awardee are capital letters, is this a problem?

Reply:
fixed